ui/page/settings/app_settings_page.go

```diff
@@ -647,6 +680,83 @@ func (pg *AppSettingsPage) HandleUserInteractions() {
 			pg.ParentWindow().ShowModal(currentPasswordModal)
 		}
 	}
+
+	if pg.backupDEX.Clicked() {
+		// Show modal asking for dex password and then reveal the seeds.
+		dexPasswordModal := modal.NewCreatePasswordModal(pg.Load).
+			EnableName(false).
+			EnableConfirmPassword(false).
+			Title(values.String(values.StrDexPassword)).
+			SetPositiveButtonCallback(func(_, password string, pm *modal.CreatePasswordModal) bool {
+				var err error
+				defer func() {
+					if err != nil {
+						pm.SetError(err.Error())
+						pm.SetLoading(false)
+					}
+				}()
+
+				dexc := pg.AssetsManager.DexClient()
+				err = dexc.Login([]byte(password))
+				if err != nil {
+					return false
+				}
+
+				seed, err := dexc.ExportSeed([]byte(password))
+				if err != nil {
+					return false
+				}
+
+				pg.showDEXSeedModal(seed)
+				return err == nil
+			})
+
+		dexPasswordModal.SetPasswordTitleVisibility(false)
+		pg.ParentWindow().ShowModal(dexPasswordModal)
+	}
+}
+
+func (pg *AppSettingsPage) handleCopyEvent(gtx C) {
+	if pg.copyDEXSeed.Clicked() {
+		clipboard.WriteOp{Text: pg.dexSeed}.Add(gtx.Ops)
+		pg.copyDEXSeed.Text = values.String(values.StrCopied)
+		pg.copyDEXSeed.Color = pg.Theme.Color.Success
+		time.AfterFunc(time.Second*3, func() {
+			pg.copyDEXSeed.Text = values.String(values.StrCopy)
+			pg.copyDEXSeed.Color = pg.Theme.Color.Primary
+			pg.ParentWindow().Reload()
+		})
+	}
+}
+
+func (pg *AppSettingsPage) showDEXSeedModal(seed dex.Bytes) {
+	defer utils.ZeroBytes(seed)
+	pg.dexSeed = stringifySeed(seed)
+	seedModal := modal.NewSuccessModal(pg.Load, values.String(values.StrDEXSeed), modal.DefaultClickFunc()).
+		UseCustomWidget(func(gtx C) D {
+			seedText := pg.Theme.Body1(pg.dexSeed)
+			seedText.Alignment = text.Middle
+			seedText.Color = pg.Theme.Color.GrayText2
+			return layout.Flex{Axis: layout.Vertical, Alignment: layout.Middle}.Layout(gtx,
+				layout.Rigid(seedText.Layout),
+				layout.Rigid(pg.copyDEXSeed.Layout),
+			)
+		})
+	pg.ParentWindow().ShowModal(seedModal)
+}
+
+func stringifySeed(seed dex.Bytes) string {
+	chunkRegex := regexp.MustCompile(`.{1,32}`) // 64 bytes, 128 hex characters.
+	chunks := chunkRegex.FindAllString(seed.String(), -1)
+
+	var seedChunks []string
+	subChunkRegex := regexp.MustCompile(`.{1,8}`)
+	for _, chunk := range chunks {
+		subChunks := subChunkRegex.FindAllString(chunk, -1)
+		seedChunks = append(seedChunks, strings.Join(subChunks, "  "))
+	}
+
+	return strings.Join(seedChunks, "\n")
 }
```

Comment:
Is this what upstream does? Seems new.

Reply:
Yh, added 6months ago just for better display: https://vscode.dev/github/decred/dcrdex/blob/expiredbonds/client/webserver/site/src/js/init.ts#L309